content/docs/setup/kubernetes/install/cni/index.md

```diff
@@ -45,30 +45,73 @@ replaces the functionality provided by the `istio-init` container.
+
+    1.  If installing via `helm template` fetch the `istio-cni` template in addition to the other Istio
+        templates fetched per the [Istio Helm installation procedure](/docs/setup/kubernetes/install/helm/#option-1-install-with-helm-via-helm-template)
+
+        {{< text bash >}}
+        $ helm fetch istio.io/istio-cni --untar --untardir $HOME/istio-fetch
+        {{< /text >}}
+
+1.  Create a namespace for the Istio control-plane components:
 
     {{< text bash >}}
-    $ helm template install/kubernetes/helm/istio --name istio --namespace istio-system \
-      --set istio_cni.enabled=true > $HOME/istio.yaml
+    $ kubectl create namespace istio-system
     {{< /text >}}
 
+1.  Install Istio CNI.
+
+    1.  Option 1.  Render and apply the Istio CNI Helm template.
+
+        {{< text bash >}}
+        $ helm template $HOME/istio-fetch/istio-cni --name=istio-cni --namespace=istio-system | kubectl apply -f -
+        {{< /text >}}
+
+    1.  Option 2.  Install via Helm and Tiller via `helm install`.
+
+        {{< text bash >}}
+        $ helm install istio.io/istio-cni --name=istio-cni --namespace=istio-system
+        {{< /text >}}
+
     {{< tip >}}
     Refer to the full set of `istio-cni` Helm parameters in the chart's [`values.yaml`](https://github.com/istio/cni/blob/master/deployments/kubernetes/install/helm/istio-cni/values.yaml).
     {{< /tip >}}
 
+1.  Add the `--set istio_cni.enabled=true` setting to the [Istio Installation with Helm procedure](/docs/setup/kubernetes/install/helm/) to include the installation of the Istio CNI plugin. For example:
+
+    {{< text bash >}}
+    $ helm template install/kubernetes/helm/istio --name istio --namespace istio-system \
+      --set istio_cni.enabled=true > $HOME/istio.yaml
+    {{< /text >}}
+
 ### Example: excluding specific Kubernetes namespaces
 
 The following example configures the Istio CNI plugin to ignore pods in the namespaces `istio-system`,
 `foo_ns`, and `bar_ns`.
 
-1.  Create an Istio manifest with the Istio CNI plugin enabled and override the default
-    configuration of the `istio-cni` Helm chart's `logLevel` and `excludeNamespaces` parameters:
+1.  Create an `istio-system` namespace for the Istio control-plane components.
 
     {{< text bash >}}
-    $ helm template install/kubernetes/helm/istio --name istio --namespace istio-system \
-    --set istio_cni.enabled=true \
+    $ kubectl create namespace istio-system
+    {{< /text >}}
+
+1.  Create and apply Istio CNI manifest.  Override the default configuration of the `istio-cni` Helm
+    chart's `logLevel` and `excludeNamespaces` parameters:
+
+    {{< text bash >}}
+    $ helm template $HOME/istio-fetch/istio-cni --name=istio-cni --namespace=istio-system \
     --set istio-cni.logLevel=info \
-    --set istio-cni.excludeNamespaces={"istio-system,foo_ns,bar_ns"} > $HOME/istio.yaml
+    --set istio-cni.excludeNamespaces={"istio-system,foo_ns,bar_ns"} > $HOME/istio-cni.yaml
```

Comment:
Do you want to keep the "istio-cni" part of "istio-cni.excludeNamespaces".  I am not sure if it breaks including it,  but it shouldn't be required.

Reply:
you're right ... will fix